validation/td-validation.ttl

```diff
@@ -32,7 +32,7 @@
                 a text for UI representation) based on a default
                 language."""^^rdf:HTML ;
         sh:nodeKind sh:Literal ;
-        sh:datatype rdf:langString ;
+        sh:or ( [ sh:datatype xsd:string ] [ sh:datatype rdf:langString ] ) ;
```

Comment:
This was apparently not correct

Reply:
I remember wrestling with something like this, maybe the comments https://github.com/w3c/wot-thing-description/pull/1684 can help @mahdanoura or @egekorkan .